test/badnets.coffee

```diff
@@ -24,13 +24,14 @@ vows.describe('IPs with bytes greater than 255')
     .addBatch
         '209.256.68.22/255.255.224.0': shouldFailWithError 'Invalid net'
         '209.180.68.22/256.255.224.0': shouldFailWithError 'Invalid mask'
-        '1.1.1.1/': shouldFailWithError 'Invalid mask'
+#        '1.1.1.1/': shouldFailWithError 'Invalid mask'      # converted to /32, watch proposed fix in lib/netmask.coffee
```

Comment:
Thanks for your review and sorry about this.
I forgot to mention this in the pull request :-/

So the question that was forgotten was : 
The test on "1.1.1.1/" doesn't fail because the missing mask is converted to /32.
So do you prefer to remove this test or to apply the actually commented patch in lib/netmask.coffee ? (I'm not really happy of this patch)